mitodl/odl-video-service
# Added server status end-point for checking application certificate

#### What are the relevant tickets?
https://github.com/mitodl/odl-video-service/issues/487

#### What's this PR do?
- It add a certificate expiry health info to odlvideo status api

#### How should this be manually tested?
- open status api `http://od.odl.local:8089/status/?token=statustoken` and check certificate info

@pdpinch 

#### Any background context you want to provide?
Blocked by release of https://github.com/mitodl/django-server-status/pull/34

#### Screenshots (if appropriate)
<img width="608" alt="screen shot 2018-09-19 at 10 26 11 pm" src="https://user-images.githubusercontent.com/10431250/45770150-08c56480-bc5b-11e8-8e5f-fc60c29f3836.png">
